nltk/parse/stanford.py

```diff
@@ -23,6 +23,7 @@
 from nltk.parse.api import ParserI
 from nltk.parse.dependencygraph import DependencyGraph
 from nltk.tree import Tree
+from nltk.parse.corenlp import CoreNLPParser, CoreNLPDependencyParser
```

Comment:
I would do this:
```python
from nltk.parse import corenlp
```

and later refer to `CoreNLPParser` and `CoreNLPDependencyParser` as `corenlp.CoreNLPParser` and `corenlp.CoreNLPDependencyParser`.